![imagen](fotos/IMAGEN_vinos-tinto.jpg)

# CALIDAD DEL VINO TINTO

## DESCRIPCION

El conjunto de datos nos da información sobre el vino tinto portugués. A partir de propiedades fisico-químicas se va a intentar determinar la calidad del vino.

## DATOS

El conjunto de datos utilizados proviene de un dataset alojado en Kaggle con el fin de determinar la calidad del vino tinto. Este conjunto de datos también está disponible en el repositorio de aprendizaje automático de la UCI, https://archive.ics.uci.edu/ml/datasets/wine+quality .

El archivo extraido de Kaggle es:

> winequality-red.csv

In [1]:
import pandas as pd

In [2]:
df = pd.read_csv ("data/winequality-red.csv", sep=",")
df

,fixed acidity,volatile acidity,citric acid,residual sugar,chlorides,free sulfur dioxide,total sulfur dioxide,density,pH,sulphates,alcohol,quality
0,7.4,0.700,0.00,1.9,0.076,11.0,34.0,0.99780,3.51,0.56,9.4,5
1,7.8,0.880,0.00,2.6,0.098,25.0,67.0,0.99680,3.20,0.68,9.8,5
2,7.8,0.760,0.04,2.3,0.092,15.0,54.0,0.99700,3.26,0.65,9.8,5
3,11.2,0.280,0.56,1.9,0.075,17.0,60.0,0.99800,3.16,0.58,9.8,6
4,7.4,0.700,0.00,1.9,0.076,11.0,34.0,0.99780,3.51,0.56,9.4,5
...,...,...,...,...,...,...,...,...,...,...,...,...
1594,6.2,0.600,0.08,2.0,0.090,32.0,44.0,0.99490,3.45,0.58,10.5,5
1595,5.9,0.550,0.10,2.2,0.062,39.0,51.0,0.99512,3.52,0.76,11.2,6
1596,6.3,0.510,0.13,2.3,0.076,29.0,40.0,0.99574,3.42,0.75,11.0,6
1597,5.9,0.645,0.12,2.0,0.075,32.0,44.0,0.99547,3.57,0.71,10.2,5


In [3]:
df.info()

<class 'pandas.DataFrame'>
RangeIndex: 1599 entries, 0 to 1598
Data columns (total 12 columns):
 #   Column                Non-Null Count  Dtype  
---  ------                --------------  -----  
 0   fixed acidity         1599 non-null   float64
 1   volatile acidity      1599 non-null   float64
 2   citric acid           1599 non-null   float64
 3   residual sugar        1599 non-null   float64
 4   chlorides             1599 non-null   float64
 5   free sulfur dioxide   1599 non-null   float64
 6   total sulfur dioxide  1599 non-null   float64
 7   density               1599 non-null   float64
 8   pH                    1599 non-null   float64
 9   sulphates             1599 non-null   float64
 10  alcohol               1599 non-null   float64
 11  quality               1599 non-null   int64  
dtypes: float64(11), int64(1)
memory usage: 150.0 KB


#### ANALIZANDO EL CONJUNTO DE DATOS:  

- 1599 ENTRADAS

- 12 COLUMNAS

- SIN VALORES NULOS

- VARIABLES NUMERICAS

COLUMNAS se dividen en:

- Variables fisicoquímicas  &rarr;  predictores:

    - acidez fija y volátil
    - acido cítrico
    - azucar residual
    - cloruros
    - dióxido de azufre libre
    - dióxido de azufre total
    - densidad
    - pH
    - sulfatos
    - alcohol

- Variable sensorial  &rarr;  objetivo:

    - calidad 

## OBJETIVO


El objetivo es modelar la calidad del vino a partir de pruebas fisicoquímicas

## AGRADECIMIENTOS

P. Cortez, A. Cerdeira, F. Almeida, T. Matos y J. Reis. Modelado de preferencias de vino mediante minería de datos a partir de propiedades fisicoquímicas. En Decision Support Systems, Elsevier, 47(4):547-553, 2009.


## BIBLIOGRAFIA

Publicación:

P. Cortez, A. Cerdeira, F. Almeida, T. Matos y J. Reis. Modelado de preferencias de vino mediante minería de datos a partir de propiedades fisicoquímicas.
En Decision Support Systems, Elsevier, 47(4):547-553, 2009.